distributor/src/distributor/messenger.bal

```diff
@@ -0,0 +1,130 @@
+import ballerina/config;
+import ballerina/log;
+import ballerina/stringutils;
+import wso2/twilio;
+
+const INVALID_NO = "Invalid no";
+
+twilio:TwilioConfiguration twilioConfig = {
+    accountSId: config:getAsString("eclk.sms.twilio.accountSid"),
+    authToken: config:getAsString("eclk.sms.twilio.authToken"),
+    xAuthyKey: config:getAsString("eclk.sms.twilio.authyApiKey")
+};
+
+twilio:Client twilioClient = new(twilioConfig);
+
+// Keeps registered sms recipients in-memory. Values are populated in every service init and recipient registration
+string[] mobileSubscribers = [];
+string sourceMobile = config:getAsString("eclk.sms.twilio.source");
+
+# Send SMS notification to all the subscribers.
+#
+# + electionCode - The respective code that represents the type of election
+# + resultCode - The predefined code for a released result
+function sendSMS(string electionCode, string resultCode) {
+    string? retrievedData = getDivision(resultCode);
+    string division = retrievedData is string ? retrievedData : resultCode;
+    string message  = "Results will be releasing soon for " + electionCode +  "/" + division + "(" + resultCode + ")";
+
+    foreach string targetMobile in mobileSubscribers {
+        if (targetMobile == INVALID_NO) {
+            continue;
+        }
+        var response = twilioClient->sendSms(sourceMobile, targetMobile, message);
```

Comment:
Should this message sending also be randomized and should the message sending maybe happen in parallel? Currently all sequential and in the same order right?

Reply:
Since this notification does not provide any result data, I kept it sequential. Do you think we should improve it to be randomized?

Reply:
It would be ice if it was parallel & random. Or at least random.